conf/messages.cy

```diff
@@ -433,7 +483,7 @@ view.beta.paragraph1=Gallwch ddefnyddio’r gwasanaeth hwn i uwchlwytho dogfenna
 view.beta.paragraph2=Mae ond angen i chi anfon dogfennau i CThEM os yw statws eich datganiad yn dangos bod rhaid anfon dogfennau.
 view.beta.paragraph3=Mae llwybrau clirio’r tollau’n cynnwys:
 view.beta.routes.route1=Llwybr 1: gwiriad llawn o’r dogfennau
-view.beta.routes.route1cap=Llwybr 1 CAP (Common Agricultural Policy)
+view.beta.routes.route1cap=Trywydd 1 CAP (Polisi Amaethyddol Cyffredin)
```

Comment:
Could we also change this one to **Llwybr** 1 CAP (Polisi Amaethyddol Cyffredin)

Reply:
Ok, I will change it. Thought Irene asked to leave it as it is..